scripts/halium

```diff
@@ -416,7 +416,7 @@ mountroot() {
 	done
 
 	# On systems with A/B partition layout, current slot is provided via cmdline parameter.
-	ab_slot_suffix=$(grep -o 'androidboot\.slot_suffix=..' /proc/cmdline |  cut -d "=" -f2)
+	export ab_slot_suffix=$(grep -o 'androidboot\.slot_suffix=..' /proc/cmdline |  cut -d "=" -f2)
```

Comment:
`export`-ing from this script seems risky. It could cause this environment to become part of the userspace init. Is that what you are trying to achieve?

Reply:
Hi, no that isn't necessarily what I want. I didn't consider that at all to be honest.

It seems like it would make more sense to make that variable accessible from everywhere in ramdisk init, rather than to repeat the command. But for this PR I only need to be able to access the slot suffix in the `mountroot` function, what do you think is the best way to approach this?

Perhaps it would make the most sense to just fetch it from cmdline again.

Reply:
I've decided to instead just fetch the variable again from cmdline where it's needed.